### ISSUE TITLE
Issue running container with nvidia drivers 470.82.01

### ISSUE THREAD

**Issue by seanrmurphy** [ehfd/docker-nvidia-glx-desktop#16]
We have encountered issues running this container with nvidia drivers 470.82.01.

The problem is with nvidia - they don't provide this driver version on `download.nvidia.com` (http://download.nvidia.com/XFree86/Linux-x86_64/) - the container breaks with a 404 when `curl`ing down the files.. Might be nice if `entrypoint.sh` printed a more clear error message in this case.

Just putting this here in case anyone else encounters this issue.

**Comment by ehfd** [comment]
That looks like a datacenter driver... could fallback to a tesla link if the download fails. Let me see what I can do.

**Comment by ehfd** [comment]
> We have encountered issues running this container with nvidia drivers 470.82.01.
> 
> The problem is with nvidia - they don't provide this driver version on `download.nvidia.com` (http://download.nvidia.com/XFree86/Linux-x86_64/) - the container breaks with a 404 when `curl`ing down the files.. Might be nice if `entrypoint.sh` printed a more clear error message in this case.
> 
> Just putting this here in case anyone else encounters this issue.

https://download.nvidia.com/XFree86/Linux-x86_64/470.82.01/
The file does seem to exist now, irrelevant of whether it is Tesla or not.

**Comment by ehfd** [comment]
Behavior:
```
user@xgl:/tmp$ DRIVER_VERSION=450.156.00
user@xgl:/tmp$ wget https://download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:31:53--  https://download.nvidia.com/XFree86/Linux-x86_64/450.156.00/NVIDIA-Linux-x86_64-450.156.00.run
Resolving download.nvidia.com (download.nvidia.com)... d.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:31:53 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:31:53--  https://download.nvidia.com/tesla/450.156.00/NVIDIA-Linux-x86_64-450.156.00.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:31:53 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:31:53--  https://http.download.nvidia.com/XFree86/Linux-x86_64/450.156.00/NVIDIA-Linux-x86_64-450.156.00.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:31:53 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:31:53--  https://http.download.nvidia.com/tesla/450.156.00/NVIDIA-Linux-x86_64-450.156.00.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:31:53 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:31:53--  https://us.download.nvidia.com/XFree86/Linux-x86_64/450.156.00/NVIDIA-Linux-x86_64-450.156.00.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:31:53 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:31:54--  https://us.download.nvidia.com/tesla/450.156.00/NVIDIA-Linux-x86_64-450.156.00.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 141312258 (135M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-450.156.00.run’

NVIDIA-Linux-x86_64-450.156.0 100%[=================================================>] 134.77M   240MB/s    in 0.6s

2021-11-23 20:31:55 (240 MB/s) - ‘NVIDIA-Linux-x86_64-450.156.00.run’ saved [141312258/141312258]
```
```
user@xgl:/tmp$ DRIVER_VERSION=460.39
user@xgl:/tmp$ wget https://download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:27:44--  https://download.nvidia.com/XFree86/Linux-x86_64/460.39/NVIDIA-Linux-x86_64-460.39.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 178310565 (170M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-460.39.run.2’

NVIDIA-Linux-x86_64-460.39.ru 100%[=================================================>] 170.05M   234MB/s    in 0.7s

2021-11-23 20:27:45 (234 MB/s) - ‘NVIDIA-Linux-x86_64-460.39.run.2’ saved [178310565/178310565]

user@xgl:/tmp$ wget https://download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:27:45--  https://download.nvidia.com/tesla/460.39/NVIDIA-Linux-x86_64-460.39.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:27:45 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:27:45--  https://http.download.nvidia.com/XFree86/Linux-x86_64/460.39/NVIDIA-Linux-x86_64-460.39.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 178310565 (170M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-460.39.run.3’

NVIDIA-Linux-x86_64-460.39.ru 100%[=================================================>] 170.05M   242MB/s    in 0.7s

2021-11-23 20:27:46 (242 MB/s) - ‘NVIDIA-Linux-x86_64-460.39.run.3’ saved [178310565/178310565]

user@xgl:/tmp$ wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:27:46--  https://http.download.nvidia.com/tesla/460.39/NVIDIA-Linux-x86_64-460.39.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:27:46 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:27:46--  https://us.download.nvidia.com/XFree86/Linux-x86_64/460.39/NVIDIA-Linux-x86_64-460.39.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 178310565 (170M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-460.39.run.4’

NVIDIA-Linux-x86_64-460.39.ru 100%[=================================================>] 170.05M   240MB/s    in 0.7s

2021-11-23 20:27:46 (240 MB/s) - ‘NVIDIA-Linux-x86_64-460.39.run.4’ saved [178310565/178310565]

user@xgl:/tmp$ wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:27:47--  https://us.download.nvidia.com/tesla/460.39/NVIDIA-Linux-x86_64-460.39.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:27:47 ERROR 404: Not Found.
```
```
user@xgl:/tmp$ DRIVER_VERSION=460.106.00
user@xgl:/tmp$ wget https://download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:28:58--  https://download.nvidia.com/XFree86/Linux-x86_64/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:28:58 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:28:58--  https://download.nvidia.com/tesla/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:28:58 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:28:58--  https://http.download.nvidia.com/XFree86/Linux-x86_64/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:28:58 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:28:58--  https://http.download.nvidia.com/tesla/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:28:59 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:28:59--  https://us.download.nvidia.com/XFree86/Linux-x86_64/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:28:59 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:29:00--  https://us.download.nvidia.com/tesla/460.106.00/NVIDIA-Linux-x86_64-460.106.00.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 179941594 (172M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-460.106.00.run’

NVIDIA-Linux-x86_64-460.106.0 100%[=================================================>] 171.61M   240MB/s    in 0.7s

2021-11-23 20:29:00 (240 MB/s) - ‘NVIDIA-Linux-x86_64-460.106.00.run’ saved [179941594/179941594]
```
```
user@xgl:/tmp$ DRIVER_VERSION=470.82.01
user@xgl:/tmp$ wget https://download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:29:57--  https://download.nvidia.com/XFree86/Linux-x86_64/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 271773846 (259M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-470.82.01.run’

NVIDIA-Linux-x86_64-470.82.01 100%[=================================================>] 259.18M   249MB/s    in 1.0s

2021-11-23 20:29:58 (249 MB/s) - ‘NVIDIA-Linux-x86_64-470.82.01.run’ saved [271773846/271773846]

user@xgl:/tmp$ wget https://download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:29:58--  https://download.nvidia.com/tesla/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:29:59 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:29:59--  https://http.download.nvidia.com/XFree86/Linux-x86_64/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 271773846 (259M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-470.82.01.run.1’

NVIDIA-Linux-x86_64-470.82.01 100%[=================================================>] 259.18M   232MB/s    in 1.1s

2021-11-23 20:30:00 (232 MB/s) - ‘NVIDIA-Linux-x86_64-470.82.01.run.1’ saved [271773846/271773846]

user@xgl:/tmp$ wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:30:00--  https://http.download.nvidia.com/tesla/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:30:00 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:30:00--  https://us.download.nvidia.com/XFree86/Linux-x86_64/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:30:00 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:30:01--  https://us.download.nvidia.com/tesla/470.82.01/NVIDIA-Linux-x86_64-470.82.01.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 271773846 (259M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-470.82.01.run.2’

NVIDIA-Linux-x86_64-470.82.01 100%[=================================================>] 259.18M   228MB/s    in 1.1s

2021-11-23 20:30:02 (228 MB/s) - ‘NVIDIA-Linux-x86_64-470.82.01.run.2’ saved [271773846/271773846]
```
```
user@xgl:/tmp$ DRIVER_VERSION=470.86
user@xgl:/tmp$ wget https://download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:34:32--  https://download.nvidia.com/XFree86/Linux-x86_64/470.86/NVIDIA-Linux-x86_64-470.86.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 273149806 (260M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-470.86.run’

NVIDIA-Linux-x86_64-470.86.ru 100%[=================================================>] 260.50M   231MB/s    in 1.1s

2021-11-23 20:34:33 (231 MB/s) - ‘NVIDIA-Linux-x86_64-470.86.run’ saved [273149806/273149806]

user@xgl:/tmp$ wget https://download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:34:33--  https://download.nvidia.com/tesla/470.86/NVIDIA-Linux-x86_64-470.86.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:34:33 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:34:33--  https://http.download.nvidia.com/XFree86/Linux-x86_64/470.86/NVIDIA-Linux-x86_64-470.86.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 273149806 (260M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-470.86.run.1’

NVIDIA-Linux-x86_64-470.86.ru 100%[=================================================>] 260.50M   254MB/s    in 1.0s

2021-11-23 20:34:34 (254 MB/s) - ‘NVIDIA-Linux-x86_64-470.86.run.1’ saved [273149806/273149806]

user@xgl:/tmp$ wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:34:34--  https://http.download.nvidia.com/tesla/470.86/NVIDIA-Linux-x86_64-470.86.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:34:34 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:34:34--  https://us.download.nvidia.com/XFree86/Linux-x86_64/470.86/NVIDIA-Linux-x86_64-470.86.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 273149806 (260M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-470.86.run.2’

NVIDIA-Linux-x86_64-470.86.ru 100%[=================================================>] 260.50M  65.5MB/s    in 4.0s

2021-11-23 20:34:38 (65.5 MB/s) - ‘NVIDIA-Linux-x86_64-470.86.run.2’ saved [273149806/273149806]

user@xgl:/tmp$ wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:34:40--  https://us.download.nvidia.com/tesla/470.86/NVIDIA-Linux-x86_64-470.86.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:34:40 ERROR 404: Not Found.
```
```
user@xgl:/tmp$ DRIVER_VERSION=495.44
user@xgl:/tmp$ wget https://download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:36:54--  https://download.nvidia.com/XFree86/Linux-x86_64/495.44/NVIDIA-Linux-x86_64-495.44.run
idia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.runResolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 282259831 (269M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-495.44.run’

NVIDIA-Linux-x86_64-495.44.ru 100%[=================================================>] 269.18M   250MB/s    in 1.1s

2021-11-23 20:36:55 (250 MB/s) - ‘NVIDIA-Linux-x86_64-495.44.run’ saved [282259831/282259831]

user@xgl:/tmp$ wget https://download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:36:55--  https://download.nvidia.com/tesla/495.44/NVIDIA-Linux-x86_64-495.44.run
Resolving download.nvidia.com (download.nvidia.com)... 192.229.210.202, 2606:2800:21f:150d:87d:2526:2d1:177d
Connecting to download.nvidia.com (download.nvidia.com)|192.229.210.202|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:36:55 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://http.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:36:55--  https://http.download.nvidia.com/XFree86/Linux-x86_64/495.44/NVIDIA-Linux-x86_64-495.44.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 282259831 (269M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-495.44.run.1’

NVIDIA-Linux-x86_64-495.44.ru 100%[=================================================>] 269.18M   247MB/s    in 1.1s

2021-11-23 20:36:56 (247 MB/s) - ‘NVIDIA-Linux-x86_64-495.44.run.1’ saved [282259831/282259831]

user@xgl:/tmp$ wget https://http.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:36:56--  https://http.download.nvidia.com/tesla/495.44/NVIDIA-Linux-x86_64-495.44.run
Resolving http.download.nvidia.com (http.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to http.download.nvidia.com (http.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:36:56 ERROR 404: Not Found.

user@xgl:/tmp$ wget https://us.download.nvidia.com/XFree86/Linux-x86_64/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:36:56--  https://us.download.nvidia.com/XFree86/Linux-x86_64/495.44/NVIDIA-Linux-x86_64-495.44.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 282259831 (269M) [application/octet-stream]
Saving to: ‘NVIDIA-Linux-x86_64-495.44.run.2’

NVIDIA-Linux-x86_64-495.44.ru 100%[=================================================>] 269.18M  64.7MB/s    in 4.2s

2021-11-23 20:37:00 (64.7 MB/s) - ‘NVIDIA-Linux-x86_64-495.44.run.2’ saved [282259831/282259831]

user@xgl:/tmp$ wget https://us.download.nvidia.com/tesla/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
--2021-11-23 20:37:01--  https://us.download.nvidia.com/tesla/495.44/NVIDIA-Linux-x86_64-495.44.run
Resolving us.download.nvidia.com (us.download.nvidia.com)... 192.229.211.70, 2606:2800:21f:3aa:dcf:37b:1ed6:1fb
Connecting to us.download.nvidia.com (us.download.nvidia.com)|192.229.211.70|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2021-11-23 20:37:01 ERROR 404: Not Found.
```

**Comment by ehfd** [comment]
Resolved with commit 952ff0c8ca3161c7f38146d775b3b9826b4dc06a

**Comment by johncadengo** [comment]
Thanks @ehfd 💯
